src/controller/data_model/controller-clusters.matter

```diff
@@ -3837,15 +3843,15 @@ client cluster Thermostat = 513 {
   }
 
   response struct GetWeeklyScheduleResponse = 0 {
-    ENUM8 numberOfTransitionsForSequence = 0;
+    INT8U numberOfTransitionsForSequence = 0;
     DayOfWeek dayOfWeekForSequence = 1;
     ModeForSequence modeForSequence = 2;
-    INT8U payload[] = 3;
+    ThermostatScheduleTransition tayload[] = 3;
```

Comment:
Nit, is this supposed to be payload or transitions?

Reply:
This should be fixed in the xml, I just first noticed it here. I will make this as resolved since this just be autogenerated

Reply:
Should be "Transitions".  Good catch; I don't know what happened there.  ;)